examples/federation-with-user/main.go

```diff
@@ -0,0 +1,123 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/selectel/iam-go"
+	"github.com/selectel/iam-go/service/federations"
+	"github.com/selectel/iam-go/service/federations/certificates"
+	"github.com/selectel/iam-go/service/roles"
+	"github.com/selectel/iam-go/service/users"
+)
+
+var (
+	// KeystoneToken
+	token          = "gAAAAA..."
+	deleteAfterRun = false
+
+	// Prefix to be added to User-Agent.
+	prefix = "iam-go"
+
+	federationName               = "federation_name"
+	federationDescription        = "federation_description"
+	updatedFederationName        = "new_federation_name"
+	updatedFederationDescription = "new_federation_description"
+
+	certificateName        = "certificate name"
+	certificateDescription = "certificate description"
+	certificateData        = "cert"
```

Comment:
With this payload error will be returned from API, so it's better to use the real certificate data here or add comment that this data is invalid and should be replaced with actual certificate's data.

Reply:
Added file `cert.crt` and filled it with the certificate example from official API (https://developers.selectel.ru/docs/control-panel/iam/), this file then is read in code